apps/devhub/templates/devhub/versions/list.html

```diff
@@ -277,6 +277,40 @@ <h3 data-tmpl="{{ _('Delete Version {version}') }}"></h3>
   </div>
   {% endif %}
 
+  {% if addon.is_listed %}
+  <div id="modal-unlist" class="modal">
+    <form method="post" action="{{ addon.get_dev_url('unlist') }}">
+      <h3>
+        {{ _('Unlist Add-on') }}
+      </h3>
+      <p>
+        {% trans update_url='https://developer.mozilla.org/Add-ons/Install_Manifests#updateURL' %}
```

Comment:
Using this url automatically redirects me to the french localized version of the page by prepending a `fr`. However, this page doesn't exist in french, so I get a 404.

Maybe a better idea would be to use the `en-US` locale, and then users will be able to change that once on the page using MDN's locale selector?